db/static/css/app.css

```diff
@@ -255,3 +255,19 @@ footer {
 .stage-notice a {
     color: #ffff00;
 }
+
+/* Stats page */
```

Comment:
nitpick: use the same section comment style as the rest of the sections in this css file.